content/2022/en/LCNC-SEC-04-Authentication-and-Secure-Communication-Failures.md

```diff
@@ -34,6 +36,15 @@ A developer uses admin credentials to create a database connection.
 They build an application that uses that connection to show data to its users.
```

Comment:
I know this is unrelated and is not part of the change, but scenario 2 doesn't really fit here, right?

Reply:
Created an issue so it's tracked on our todo here: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/issues/67